### PR TITLE
Draft:Upgrading splunk usf version

### PR DESCRIPTION
**Pull request by mananpatel26**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DTSPO-17050

### Change description ###
Upgrading splunk usf version to 9.2.1

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by narinder17 — APPROVED** [review]

File: scripts/bootstrap_vm.sh [hunk 1]
```diff
@@ -123,83 +123,94 @@ check_download_url () {
...
 then
-  install_splunk_uf "${UF_USERNAME}" "${UF_PASSWORD}" "${UF_PASS4SYMMKEY}" "${UF_GROUP}"
+  install_splunk_uf "hmcts_soc_admin" "BNFJN0XWmkCJw0S78HLO" "$7$oNroXETNQ9CvUjVvOv+gF4vnXZMUcOV5AYqzGX8Dl4QPUgp1xrUu68laynlhotZaNcOhCQ/FXXYtSH1VehZ5YA==" "hmcts_forwarders"
 fi
 
-if [ "${NESSUS_INSTALL}" = "true" ]
+if [ "true" = "true" ]
 then
-  install_nessus "${NESSUS_SERVER}" "${NESSUS_KEY}" "${NESSUS_GROUPS}"
+  install_nessus "nessus-scanners-prod000005.platform.hmcts.net" "86395175d0e6cf61a8915d84180ed64aa8e0e0ce38e52c64fd4bc8cb29418aea" "Prod-test"
```
Comment (thomast1906) [inline scripts/bootstrap_vm.sh]: why is the key in plain text?

File: scripts/bootstrap_vm.sh [hunk 2]
```diff
@@ -123,83 +123,94 @@ check_download_url () {
...
 }
 
 # Exit on error
 set +e
 
-if [ "${UF_INSTALL}" = "true" ]
+if [ "true" = "true" ]
 then
-  install_splunk_uf "${UF_USERNAME}" "${UF_PASSWORD}" "${UF_PASS4SYMMKEY}" "${UF_GROUP}"
+  install_splunk_uf "hmcts_soc_admin" "BNFJN0XWmkCJw0S78HLO" "$7$oNroXETNQ9CvUjVvOv+gF4vnXZMUcOV5AYqzGX8Dl4QPUgp1xrUu68laynlhotZaNcOhCQ/FXXYtSH1VehZ5YA==" "hmcts_forwarders"
```
Comment (thomast1906) [inline scripts/bootstrap_vm.sh]: are they not secrets?

Comment (narinder17) [reply scripts/bootstrap_vm.sh]: reverted back to the original code to remove plain text

File: scripts/bootstrap_vm.sh (hunk 1 above)
Comment (narinder17) [reply scripts/bootstrap_vm.sh]: reverted back to the original code to remove plain text

File: scripts/bootstrap_vm.sh [hunk 3]
```diff
@@ -123,83 +123,94 @@ check_download_url () {
...
 set +e
 
-if [ "${UF_INSTALL}" = "true" ]
+if [ "true" = "true" ]
 then
   install_splunk_uf "${UF_USERNAME}" "${UF_PASSWORD}" "${UF_PASS4SYMMKEY}" "${UF_GROUP}"
 fi
 
-if [ "${NESSUS_INSTALL}" = "true" ]
+if [ "true" = "true" ]
```
Comment (madjava) [inline scripts/bootstrap_vm.sh:213]: logic needs to be reverted i believe

File: scripts/bootstrap_vm.sh [hunk 4]
```diff
@@ -123,83 +123,94 @@ check_download_url () {
...
     /opt/nessus_agent/sbin/nessuscli agent link --key=$KEY --groups=$GROUPS --host=$SERVER --port=8834
-fi
+  fi
 }
 
 # Exit on error
 set +e
 
-if [ "${UF_INSTALL}" = "true" ]
+if [ "true" = "true" ]
```
Comment (madjava) [inline scripts/bootstrap_vm.sh:208]: Here as well

**Comment by madjava** [comment]
Lots of code formatting in the `scripts/bootstrap_vm.sh` file. Do we need the formatting? id say revet em all if No

**Comment by ConnorOKane-Kainos** [comment]
Reopening via another branch to make the fixes and remove this config.

**Pull request closed without merging**